src/services/auth/reset.ts

```diff
@@ -1,37 +1,36 @@
-import { User } from '../../sequelize'
-import { resetEmail } from '../../helpers/nodemailer'
+import { User } from '../../sequelize';
+import { resetEmail } from '../../helpers/nodemailer';
 
-export default function reset(req, res, next) {
+export default function reset(req: any, res: any, next: any): void {
+  const { email } = req.body;
 
-  let { email } = req.body
+  User.findOne({ where: { email } })
```

Comment:
I want to keep in a `servieces` functions that are `express middlewares` to keep this code reusable. All functions will have parameters `req, res, next` and will return `void`.